resources/languages/en_US.ini

```diff
@@ -84,7 +84,7 @@ microgame.success="§aSUCCESS"
 microgame.failed="§cFAILED"
 microgame.worth="§l§eThis round was worth §6{%points} §epoints."
 microgame.felloffplatform="§cYou fell off the platform!"
-microgame.threwoffstage="§cYou failed because you threw yourself off the stage! :O"
+microgame.threwoffstage="§cYou failed because you throw yourself off the stage! :O"
```

Comment:
As far as I understand "threw" is correct, besides that is what Minerware's version of Cubecraft says.